resources/assets/components/CampaignSingle/index.js

```diff
@@ -55,32 +66,21 @@ class CampaignSingle extends React.Component {
     });
   }
 
-  // Make API call to GET /posts to get posts by filtered status.
-  getPostsByStatus(status, campaignId) {
+  // Make API call to GET /posts to get posts by filtered status and/or tag(s).
+  getPostsByFilter(filters) {
     this.setState({ loadingNewPosts: true });
 
-    this.api.get('posts', {
-      filter: {
-        status: status,
-        campaign_id: campaignId,
-      },
-      include: 'signup,siblings',
-    })
-    .then(json => {
-      this.setState({loadingNewPosts: false });
-      this.props.setNewPosts(json);
-    });
-  }
+    let apiFilter = {
+      campaign_id: filters.campaignId,
+      status: filters.status,
+    };
 
-  // Make API call to GET /posts to get posts by filtered tag.
-  getPostsByTag(tagSlug, campaignId) {
-    this.setState({ loadingNewPosts: true });
+    if (filters.tags.length > 0) {
+      apiFilter['tag'] = filters.tags.toString();
+    }
 
-    this.api.get('posts', {
-      filter: {
-        tag: tagSlug,
-        campaign_id: campaignId,
-      },
+    this.api.get('api/v2/posts', {
```

Comment:
We have to make the call to the `GET /posts` web route so that the request is authenticated properly and we get back all the information we need. 

The reason we were seeing issues with user information not showing up was because using `api/v2/posts` was making unauthenticated requests to northstar which doesn't return full user objects